lms/djangoapps/api_manager/courses/tasks.py

```diff
@@ -0,0 +1,16 @@
+import sys
+
+from celery import task
+from django.conf import settings
+from django.core.cache import cache
+
+@task(name=u'lms.djangoapps.api_manager.courses.tasks.cache_static_tab_content')
+def cache_static_tab_content(cache_key, html):
+    """
+    Caches course static tab contents.
+    """
+    cache_expiration = getattr(settings, 'STATIC_TAB_CONTENT_CACHE_MAX_AGE', 0)
```

Comment:
I think we shoud remove `STATIC_TAB_CONTENT_CACHE_MAX_AGE` and  `STATIC_TAB_CONTENT_CACHE_MAX_SIZE_LIMIT` in common.py and set the defaults here like this 
`cache_expiration = getattr(settings, 'STATIC_TAB_CONTENT_CACHE_MAX_AGE', 60 * 5)`